test/jdk/java/awt/Frame/FrameLocation/FrameLocation.java

```diff
@@ -22,38 +22,46 @@
  */
 
 /*
-  @test
-  @key headful
-  @bug 6895647
-  @summary X11 Frame locations should be what we set them to
-  @author anthony.petrov@oracle.com: area=awt.toplevel
-  @run main FrameLocation
+ * @test
+ * @key headful
+ * @bug 6895647
+ * @summary X11 Frame locations should be what we set them to
+ * @run main FrameLocation
  */
 
-import java.awt.*;
+import java.awt.EventQueue;
+import java.awt.Frame;
 
 public class FrameLocation {
     private static final int X = 250;
     private static final int Y = 250;
+    private static Frame f;
+    private static volatile int xPos;
+    private static volatile int yPos;
 
-    public static void main(String[] args) {
-        Frame f = new Frame("test");
-        f.setBounds(X, Y, 250, 250); // the size doesn't matter
-        f.setVisible(true);
+    public static void main(String[] args) throws Exception {
+        EventQueue.invokeAndWait(() -> {
+            f = new Frame("Frame Location Test");
+            f.setBounds(X, Y, 250, 250); // the size doesn't matter
+            f.setVisible(true);
+        });
 
         for (int i = 0; i < 10; i++) {
             // 2 seconds must be enough for the WM to show the window
             try {
                 Thread.sleep(2000);
-            } catch (InterruptedException ex) {
+            } catch (InterruptedException _) {
             }
```

Comment:
What's the point of this change? To make backporting the changeset more complicated?

Reply:
If you still prefer to leave `Thread.sleep`, you can safely remove `try`-`catch` around it because `main` is declared to throw an exception.